src/ui/components/RightBar.css

```diff
@@ -47,3 +78,24 @@
     height: calc(100vh - 170px);
   }
 }
+
+.BarImg {
+  transition: 0.2s;
```

Comment:
While the circle around the icon transitions properly, the image color itself does not. Either make both elements do it, or neither.